tests/src/test/java/IOTests.java

```diff
@@ -3,29 +3,32 @@
 import mindustry.game.*;
 import mindustry.io.*;
 import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
```

Comment:
Use wildcard imports here ([CONTRIBUTING.MD](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md#follow-the-formatting-guidelines)).

Reply:
Done